src/orders/views/OrderDetails/OrderDraftDetails/index.tsx

```diff
@@ -0,0 +1,194 @@
+import { WindowTitle } from "@saleor/components/WindowTitle";
+import { DEFAULT_INITIAL_SEARCH_DATA } from "@saleor/config";
+import useNavigator from "@saleor/hooks/useNavigator";
+import useUser from "@saleor/hooks/useUser";
+import { OrderDiscountProvider } from "@saleor/products/components/OrderDiscountProviders/OrderDiscountProvider";
+import { OrderLineDiscountProvider } from "@saleor/products/components/OrderDiscountProviders/OrderLineDiscountProvider";
+import useCustomerSearch from "@saleor/searches/useCustomerSearch";
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { customerUrl } from "../../../../customers/urls";
+import { getStringOrPlaceholder, maybe } from "../../../../misc";
+import { productUrl } from "../../../../products/urls";
+import OrderDraftCancelDialog from "../../../components/OrderDraftCancelDialog/OrderDraftCancelDialog";
+import OrderDraftPage from "../../../components/OrderDraftPage";
+import OrderProductAddDialog from "../../../components/OrderProductAddDialog";
+import OrderShippingMethodEditDialog from "../../../components/OrderShippingMethodEditDialog";
+import { useOrderVariantSearch } from "../../../queries";
+import { OrderUrlQueryParams } from "../../../urls";
+import { orderDraftListUrl } from "../../../urls";
+
+interface OrderDraftDetailsProps {
+  id: string;
+  params: OrderUrlQueryParams;
+  loading: any;
+  data: any;
+  orderAddNote: any;
+  orderLineUpdate: any;
+  orderLineDelete: any;
+  orderShippingMethodUpdate: any;
+  orderLinesAdd: any;
+  orderDraftUpdate: any;
+  orderDraftCancel: any;
+  orderDraftFinalize: any;
+  openModal: any;
+  closeModal: any;
+}
+
+export const OrderDraftDetails: React.FC<OrderDraftDetailsProps> = ({
+  id,
+  params,
+  loading,
+  data,
+  orderAddNote,
+  orderLineUpdate,
+  orderLineDelete,
+  orderShippingMethodUpdate,
+  orderLinesAdd,
+  orderDraftUpdate,
+  orderDraftCancel,
+  orderDraftFinalize,
+  openModal,
+  closeModal
+}) => {
+  const order = data.order;
+  const navigate = useNavigator();
+  const { user } = useUser();
+
+  const {
+    loadMore,
+    search: variantSearch,
+    result: variantSearchOpts
+  } = useOrderVariantSearch({
+    variables: DEFAULT_INITIAL_SEARCH_DATA
+  });
+
+  const {
+    loadMore: loadMoreCustomers,
+    search: searchUsers,
+    result: users
+  } = useCustomerSearch({
+    variables: DEFAULT_INITIAL_SEARCH_DATA
+  });
+
+  const intl = useIntl();
+
+  return (
+    <>
+      <WindowTitle
+        title={intl.formatMessage(
+          {
```

Comment:
Could be in defineMessages

Reply:
Would agree but this is the only occurance in each of 3 new components. I can still do it but IMO not worth it in that case.